aggregator/run_with_remote_cluster.sh

```diff
@@ -1,39 +1,20 @@
 #!/usr/bin/env bash
 set -e
-
 AWS_CLI_PROFILE="concepts-aggregator-local"
 CATALOGUE_DEVELOPER_ARN="arn:aws:iam::756629837203:role/catalogue-developer"
 ROOT=$(git rev-parse --show-toplevel)
 
 CONCEPTS_PIPELINE_DATE=$1
-if [ "$CONCEPTS_PIPELINE_DATE" = "" ]; then
+# If the "date" is less than 10 chars, then it is probably a Work Id
+# and someone forgot to add the date
+if [ "$CONCEPTS_PIPELINE_DATE" = "" ] ||  [ ${#CONCEPTS_PIPELINE_DATE} -lt 10 ] ; then
     echo "You must provide a pipeline date!"
     exit 1
 fi
 shift
 
-aws configure set region eu-west-1 --profile $AWS_CLI_PROFILE
-aws configure set role_arn $CATALOGUE_DEVELOPER_ARN --profile $AWS_CLI_PROFILE
-aws configure set source_profile default --profile $AWS_CLI_PROFILE
-
-function get_es_secret () {
-  aws secretsmanager get-secret-value \
-    --secret-id "$1" \
-    --profile "$AWS_CLI_PROFILE" \
-    --output text \
-    --query 'SecretString'
-}
-
-ES_HOST=$(get_es_secret "elasticsearch/concepts-$CONCEPTS_PIPELINE_DATE/public_host")
-ES_PORT=9243
-ES_SCHEME="https"
-ES_USERNAME="aggregator"
-ES_PASSWORD=$(get_es_secret "elasticsearch/concepts-$CONCEPTS_PIPELINE_DATE/aggregator/password")
-
-(cd $ROOT && docker compose run \
-  --env es_host="$ES_HOST" \
-  --env es_port="$ES_PORT" \
-  --env es_scheme="$ES_SCHEME" \
-  --env es_username="$ES_USERNAME" \
-  --env es_password="$ES_PASSWORD" \
-  aggregator "$@")
+AWS_PROFILE=catalogue-developer
```

Comment:
we've tried to move away from assuming roles like this as it's dependent on local config (although it is extremely convenient) - creating a new profile is something I copied from the `run_terraform` scripts but I don't know if there might be another simpler way that's still portable. I know @alexwlchan has opinions too...